### PR TITLE
feat: add `CalledElement#propagateAllParentVariables`

### LINKED ISSUE #50: Add `CalledElement#propagateAllParentVariables` property
The corresponding XML:
```
<bpmn:callActivity id="CallActivity_1">
  <bpmn:extensionElements>
    <zeebe:calledElement propagateAllParentVariables="true|false" />
  </bpmn:extensionElements>
</bpmn:callActivity>
```

Default value is `true`

### PR DESCRIPTION
**Pull request by romansmirnov**
closes #50

### CODE REVIEW

**Review by nikku — APPROVED** [review]

**Comment by nikku** [comment]
Thanks for your contribution.

File: resources/zeebe.json [hunk 1]
```diff
@@ -240,6 +240,11 @@
           "name": "propagateAllChildVariables",
           "isAttr": true,
           "type": "Boolean"
+        },
+        {
+          "name": "propagateAllParentVariables",
+          "isAttr": true,
+          "type": "Boolean"
```
Comment (nikku) [inline resources/zeebe.json:247]:
```suggestion
          "type": "Boolean",
          "default": true
        }
```

Defaults won't be serialized, and if you decide to go this route, please add respective read tests, too.

**Comment by nikku** [comment]
As a follow-up you _could_ decide to explicitly encode the default (`true`). This ensures only `propagateAllChildVariables=false` is serialized.

**Comment by romansmirnov** [comment]
@nikku, thanks for the follow-up. I applied the requested adjustments in a separate PR: https://github.com/camunda/zeebe-bpmn-moddle/pull/52

**Comment by nikku** [comment]
Released as [v1.0.0](https://github.com/camunda/zeebe-bpmn-moddle/blob/master/CHANGELOG.md#100), thanks!

**Pull request merged**